tw-tkms-test-starter/build.gradle

```diff
@@ -15,13 +15,14 @@ dependencies {
     annotationProcessor libraries.spotbugsAnnotations
 
     compileOnly libraries.springBootConfigurationProcessor
-
+    compileOnly libraries.javaxValidationApi
```

Comment:
I can't see the validation-api being used in this project. Are the dependencies even needed?

Reply:
Removed.